tests/unit_tests/test_wave_02.py

```diff
@@ -33,8 +33,13 @@ def test_get_no_matching_items_by_category():
     )
 
     items = vendor.get_by_category("electronics")
+    assert items == []
+    assert item_a not in items
+    assert item_b not in items
+    assert item_c not in items
```

Comment:
These asserts are not needed! Since the first assert tells us there is nothing in the list, we do not need to check if these individual items are in the list.